server/embed/config_tracing.go

```diff
@@ -40,13 +40,19 @@ func validateTracingConfig(samplingRate int) error {
 	return nil
 }
 
-func setupTracingExporter(ctx context.Context, cfg *Config) (exporter tracesdk.SpanExporter, options []otelgrpc.Option, err error) {
-	exporter, err = otlptracegrpc.New(ctx,
+type tracingExporter struct {
+	exporter tracesdk.SpanExporter
+	opts     []otelgrpc.Option
+	shutdown func(ctx context.Context)
```

Comment:
The name `shutdown` is a little strange to me, if `tracingExporter` knows what it should do on close, why not to get the `traceProvider` added to the struct directly? Two choice:
1. Add the `traceProvider` into the struct directly, and call `te.traceProvider.Shutdown(ctx)` directly on Close. 
2. Rename to `cleanup` so that the logic is agnostic to `tracingExporter`, instead it's just doing the cleanup task it's told.

Reply:
Thank you for the suggestion. I went with the first option